_includes/general/nav.html

```diff
@@ -30,5 +30,6 @@ <h3>{{ item.title }}</h3>
       {% endfor %}
       {% endif %}
     </ul>
+    <span style="color:#696969;font-size:12px">This documentation is <a href="https://creativecommons.org/licenses/by-sa/4.0/deed.en">CC BY-SA</a></span>
```

Comment:
Please DO NOT hardcode css here. Generate a class/id and put it into a CSS file.